guix-systole/packages/itk.scm

```diff
@@ -1,111 +1,132 @@
 (define-module (guix-systole packages itk)
-    #:use-module (gnu packages image-processing)
-    #:use-module (guix build-system cmake)
-    #:use-module (guix download)
-    #:use-module (guix packages)
-    )
+  #:use-module (gnu packages image-processing)
+  #:use-module (guix build-system cmake)
+  #:use-module (guix build-system trivial)
+  #:use-module (guix download)
+  #:use-module (guix git-download)
+  #:use-module ((guix licenses)
+                #:prefix license:)
+  #:use-module (guix packages))
 
 (define-public itk-slicer
   (package
     (inherit insight-toolkit)
     (version "5.4.0")
-    (source
-     (origin
-       (method url-fetch)
-       (uri
-             "https://github.com/Slicer/ITK/archive/29b78d73c81d6c00c393416598d16058704c535c.tar.gz")
-       (sha256(base32 "1cqy2rzcskfjaszww4isp6g2mg4viqcp7qacfvrc97pq1qvrs5lb"))))
+    (source (origin
+              (method url-fetch)
+              (uri
+               "https://github.com/Slicer/ITK/archive/29b78d73c81d6c00c393416598d16058704c535c.tar.gz")
+              (sha256
+               (base32
+                "1cqy2rzcskfjaszww4isp6g2mg4viqcp7qacfvrc97pq1qvrs5lb"))))
     (arguments
-      `( #:tests? #f        ; tests require network access and external data
-        #:configure-flags
-        (list  
-            "-DITK_USE_SYSTEM_GOOGLETEST=OFF"
-            "-DITK_USE_GPU=ON"
-            "-DITK_USE_SYSTEM_LIBRARIES=ON"
-            "-DITK_BUILD_SHARED=ON"
-            ;; This prevents "GTest::GTest" from being added to the 
-            ;; ITK_LIBRARIES variable in the installed CMake files. This is
-            ;; necessary as other packages using insight-toolkit could not be
-            ;; configured otherwise.
-            "-DGTEST_ROOT=gtest"
+     `(#:tests? #f ;tests require network access and external data
+       #:configure-flags (list "-DITK_USE_SYSTEM_GOOGLETEST:BOOL=OFF"
+                          "-DITK_USE_GPU:BOOL=OFF"
+                          "-DITK_USE_SYSTEM_LIBRARIES:BOOL=ON"
+                          ;; "-DITK_BUILD_SHARED:BOOL=ON"
```

Comment:
`ITK_BUILD_SHARED` should be `ON`. We don't want to re-build all packages depending on ITK under minor changes of the library